leases/services/invoice.py

```diff
@@ -270,6 +287,32 @@ def send_invoices(self) -> None:  # noqa: C901
         finally:
             self.email_admins(exited_with_errors)
 
+    def resend_failed_invoices(self, profiles: dict) -> None:
+        logger.debug("Resending failed invoices")
+        orders = self.get_failed_orders(self.season_start)
+
+        for order in orders:
+            order.set_status(
+                OrderStatus.WAITING,
+                comment=f"{get_ts()}: {_('Cleanup the invoice to attempt resending')}\n",
+            )
+
+            if not order.customer_email and not order.customer_phone:
```

Comment:
Should always update, in case email in profile has been updated and order.customer_email has not?